.azure-pipelines.yml

```diff
@@ -526,8 +526,9 @@ jobs:
           cp build/parsec*-setup.exe dist
         displayName: Build NSIS setup
         workingDirectory: $(Agent.TempDirectory)/winbuild
-      - publish: $(Agent.TempDirectory)/winbuild/dist
-        artifact: win_$(vs.arch)_installer
+      # Cannot do the NSIS installer part in CI given it requires to sign `parsec.exe`
+      - publish: $(Agent.TempDirectory)/winbuild/build/parsec-*
+        artifact: win_$(vs.arch)_installer_content
```

Comment:
Then the `makensis.exe installer.nsi` command is not necessary anymore? Should we remove it?

Reply:
I forgot to remove the nsis stuff, done.